fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/FilteredAggregationMaker.java

```diff
@@ -66,7 +66,7 @@ protected LogicalMetric makeInner(String metricName, List<String> dependentMetri
                 new TemplateDruidQuery(
                         ImmutableSet.of(filteredAggregation),
                         Collections.emptySet(),
-                        sourceMetric.getTemplateDruidQuery().getInnerQuery()
+                        sourceMetric.getTemplateDruidQuery().getInnerQuery().orElse(null)
```

Comment:
Maybe other expert on `Optional` can correct me. But isn't the point of using optional is to avoid null check? `orElse()` provides a chance to supply a default value to a nullable object then wouldn't feeding it with a null counteract the merit of `Optional`?